dist/src/main/config/gateway.yaml.template

```
@@ -224,6 +224,19 @@
       # This setting can also be overridden using the environment variable ZEEBE_GATEWAY_LONGPOLLING_ENABLED.
       # enabled: true
 
+    # multiTenancy:
+      # Enables multi tenancy for the gateway.
+      # When enabled, the gateway enhances requests with the authorized tenant ids of the requester.
+      # It also rejects requests that specify a tenant id that the requester is not authorized to access.
+      # When disabled, the gateway enhances requests with the default tenant id.
```

:wrench: Same as above, might be good to document the value of the default tenant id, i.e. `<default>` so that users are aware of it.

```suggestion
      # When disabled, the gateway enhances requests with the <default> tenant id.
```